seml/queue_experiments.py

```diff
@@ -280,6 +309,18 @@ def queue_experiments(config_file, force_duplicates, no_hash=False):
     collection = db_utils.get_collection(seml_config['db_collection'])
     configs = generate_configs(experiment_config)
 
+    batch_id = db_utils.get_max_value(collection, "batch_id")
+    if batch_id is None:
+        batch_id = 1
+    else:
+        batch_id = batch_id + 1
+
+    if "project_root_dir" not in seml_config:
+        print("Warning: 'project_root_dir' not defined in seml config. Source files will not be uploaded.")
```

Comment:
Use `logging.warning` for this.

Reply:
fixed in 9d7f9f2

Reply:
You don't need to write "Warning" if you use `logging.warning`.

Reply:
Good point. :)

Reply:
006e7d7